inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -920,9 +943,9 @@ cause_of_death_sex <- survey_design_sex %>%
 The gender specific mortality rate was `r cause_of_death_sex %>% slice(sex == "Female") %>% pull(mortality) ` deaths/10,000 persons/day amongst females and
 `r cause_of_death_sex %>% slice(sex == "Male") %>% pull(mortality) `deaths/10,000 persons/day amongst males.
 
-Reported causes of death and cause-specific mortality rates, by age, weighted
+Reported causes of death and cause-specific mortality rates, by age group, weighted
 
-```{r, weighted_death_cause_by_age}
+```{r weighted_death_cause_by_age}
```

Comment:
@zkamvar two issues here: 
1) Cant rename columns because % (95%CI) is duplicated, so throws an error. (cause_of_death as a column header looks little silly in a report...) 
2) The age groups factor levels are being ignore - presumable because you need to rename as a character (it goes 0-2, 15-29, 3-14, 30-44, 45+)

Reply:
> Cant rename columns because % (95%CI) is duplicated, so throws an error; same for gender below (cause_of_death as a column header looks little silly in a report...)

:man_facepalming: Th...this is exactly what y'all wanted and now it's blown up. I'll write a wrapper.

> The age groups factor levels are being ignore - presumable because you need to rename as a character (it goes 0-2, 15-29, 3-14, 30-44, 45+)

I'll fix it. It's due to the fact that we needed to combine character and factor columns.